xml/System.Net.Http/HttpResponseMessage.xml

```diff
@@ -484,9 +484,21 @@
         <ReturnType>System.Net.Http.Headers.HttpResponseHeaders</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the collection of trailing headers included in a HTTP response.</summary>
+        <value>The collection of trailing headers in the HTTP response.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+Forbidden headers will be ignored.
+
+Since trailing headers are send as a HTTP header appended at the end of the response message; the <xref:System.Net.Http.HttpResponseMessage.TrailingHeaders%2A> property will return an empty HttpResponseHeaders instance if its accessed and the response content has not been read completely (See example below). 
```

Comment:
```suggestion
Since trailing headers are send as an HTTP header appended at the end of the response message, the `TrailingHeaders` property returns an empty <xref:System.Net.Http.Headers.HttpResponseHeaders> instance if it is accessed and the response content has not been read completely, as the following example shows. 
```
Were you planning to include an examples section for the example (in which case it won't follow) or to include in the Remarks after this paragraph (in which case it does follow)?

Reply:
I was planning to include it as a code snippet (under #2734) right after this paragraph. So it does follow I think.

Reply:
I will remove `as the following example shows` and I will add it back on #2734.